src/main/java/de/craften/ui/swingmaterial/MaterialColor.java

```diff
@@ -279,6 +279,7 @@
     public static final Color GREY_600 = Color.decode("#757575");
     public static final Color GREY_700 = Color.decode("#616161");
     public static final Color GREY_800 = Color.decode("#424242");
+    public static final Color GREY_850 = Color.decode("#313131");
```

Comment:
Okay, let's keep `GREY_850` then. Do you want to change `MaterialWindow.light()` to use `GREY_50`?